test/unit/server/lib/image/gravatar.test.js

```diff
@@ -2,23 +2,51 @@ const should = require('should');
 const Gravatar = require('../../../../../core/server/lib/image/gravatar');
 
 describe('lib/image: gravatar', function () {
+    const gravatarUrl = 'https://www.gravatar.com/avatar/{hash}?s={size}&r={rating}&d={_default}';
+
+    it('can build a gravatar url', function () {
+        const gravatar = new Gravatar({config: {
+            isPrivacyDisabled: () => false,
+            get: (config) => {
+                return config === 'gravatar' ? {
+                    url: gravatarUrl
+                } : null;
+            }
+        }, request: () => {}});
+
+        gravatar.url('exists@example.com', {
+            size: 180,
+            rating: 'r'
+        }).should.eql('https://www.gravatar.com/avatar/ef6dcde5c99bb8f685dd451ccc3e050a?s=180&r=r&d=blank');
+    });
+
     it('can successfully lookup a gravatar url', function (done) {
         const gravatar = new Gravatar({config: {
-            isPrivacyDisabled: () => false
+            isPrivacyDisabled: () => false,
+            get: (config) => {
+                return config === 'gravatar' ? {
+                    url: gravatarUrl
+                } : null;
+            }
         }, request: () => {}});
 
         gravatar.lookup({email: 'exists@example.com'}).then(function (result) {
             should.exist(result);
             should.exist(result.image);
-            result.image.should.eql('//www.gravatar.com/avatar/ef6dcde5c99bb8f685dd451ccc3e050a?s=250&d=mm&r=x');
+            result.image.should.eql('https://www.gravatar.com/avatar/ef6dcde5c99bb8f685dd451ccc3e050a?s=250&r=x&d=mp');
```

Comment:
I'm not sure what impact changing from no protocol to https will have and whether it matters.

Reply:
Looks like this change is actually preferable 🌮 https://github.com/TryGhost/Ghost/issues/11095